apps/subscribers/models/subscriber.py

```diff
@@ -0,0 +1,15 @@
+import uuid
+
+from django.db import models
+
+
+class Subscriber(models.Model):
+    # TODO: Add base model and inherit from that
+    subscriber_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    endpoint_url = models.CharField(unique=True, max_length=300, blank=False, default=None)
```

Comment:
Django creates id column for every model by default. We don't need to create a column for that.